pagination/cursor/README.md

```diff
@@ -0,0 +1,57 @@
+## Cursor-based Pagination
```

Comment:
Might be better not to use "cursor", has the potential to confuse with rest "next cursor"